packages/board-server/Dockerfile

```diff
@@ -1,11 +1,34 @@
-FROM node:20
-COPY --from=breadboard . /breadboard/
-# For the time being, we will assume container deployments are exclusively sqlite.
-ENV STORAGE_BACKEND "sqlite"
-ENV GOOGLE_CLOUD_PROJECT "n/a"
-ENV ALLOWED_ORIGINS ""
-WORKDIR /breadboard
+# Build stage
+FROM node:20 AS build
```

Comment:
Not a big deal, just curious. Can we use slim for both? I remember that slim doesn't have git, but we're not using that, is there something else that our build process needs?

Reply:
Thanks Al, yes we can, I just forgot to update it in the first stage